### PR TITLE
chore: cleanup repo

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: tests/lib/rules/spell-kents-name-correctly.js [hunk 1]
```diff
@@ -18,73 +18,31 @@ const rule = require('../../../lib/rules/spell-kents-name-correctly'),
 const ruleTester = new RuleTester()
 ruleTester.run('spell-kents-name-correctly', rule, {
   valid: [
-    // give me some code that won't trigger a warning
-  ],
+    `Kent`,
+    `Kent C.`,
+    // `Kent C. Dodds`,
```
Comment (MichaelDeBoey) [inline tests/lib/rules/spell-kents-name-correctly.js:23]: This line is failing apparently

Logged a bug: https://github.com/kentcdodds/eslint-plugin-kentcdodds/issues/2

Comment (MichaelDeBoey) [reply tests/lib/rules/spell-kents-name-correctly.js:23]: https://github.com/kentcdodds/eslint-plugin-kentcdodds/issues/2

**Review by kentcdodds — APPROVED** [review]
Awesome! Thanks :)

**Comment by kentcdodds** [comment]
@all-contributors please add @MichaelDeBoey for docs, code, and tests

**Pull request merged**